cw/logger.py

```diff
@@ -0,0 +1,10 @@
+import logging
```

Comment:
defining the logger in cw/__init__.py created a circular dependency problem.  cw/__init__.py imported the spark module, which would either define its own logger, or import the logger from cw/__init__.py.  in its own namespace, can be imported anywhere its needed